2025-06-07 05:58:49 - main - INFO - VKBot: инициализация…
2025-06-07 05:58:49 - main - INFO - VKBot: инициализирован.
2025-06-07 05:58:49 - main - INFO - VKBot: Long Poll старт…
2025-06-07 05:58:49 - main - INFO - MAIN_APP: VKBot запущен
2025-06-07 05:58:49 - main - INFO - TelegramBot: инициализация…
2025-06-07 05:58:50 - main - INFO - TelegramBot: хендлеры добавлены
2025-06-07 05:58:50 - main - INFO - MAIN_APP: TelegramBot запущен
2025-06-07 05:58:50 - main - INFO - TelegramBot: запускаем polling…
2025-06-07 05:58:51 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7649614124:AAFyK56ESGm79ivpMd-kyXjFPmb7fQNyM-w/getMe "HTTP/1.1 200 OK"
2025-06-07 05:58:51 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7649614124:AAFyK56ESGm79ivpMd-kyXjFPmb7fQNyM-w/deleteWebhook "HTTP/1.1 200 OK"
2025-06-07 05:58:51 - telegram.ext.Application - INFO - Application started
2025-06-07 05:58:58 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7649614124:AAFyK56ESGm79ivpMd-kyXjFPmb7fQNyM-w/getUpdates "HTTP/1.1 200 OK"
2025-06-07 05:58:59 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7649614124:AAFyK56ESGm79ivpMd-kyXjFPmb7fQNyM-w/sendMessage "HTTP/1.1 200 OK"
2025-06-07 05:59:09 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7649614124:AAFyK56ESGm79ivpMd-kyXjFPmb7fQNyM-w/getUpdates "HTTP/1.1 200 OK"
2025-06-07 05:59:11 - main - INFO - MAIN_APP: Остановка ботов…
2025-06-07 05:59:11 - main - INFO - TelegramBot: получен сигнал остановки
2025-06-07 05:59:11 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-17' coro=<Application.shutdown() done, defined at C:\Users\denmi\AppData\Roaming\Python\Python312\site-packages\telegram\ext\_application.py:515> exception=RuntimeError('This Application is still running!')>
Traceback (most recent call last):
  File "C:\Users\denmi\AppData\Roaming\Python\Python312\site-packages\telegram\ext\_application.py", line 534, in shutdown
    raise RuntimeError("This Application is still running!")
RuntimeError: This Application is still running!
2025-06-07 05:59:11 - main - INFO - VKBot: получен сигнал остановки
